express.js

```diff
@@ -6,7 +6,9 @@ const cookieParser = require('cookie-parser')
 
 const app = express()
 
-app.use('/docs', express.static('docs'))
+app.use('/docs', express.static('docs', {
```

Comment:
I must make use of this express endpoint more. I keep using the preview on vscode and its actually really annoying